google/auth/_cloud_sdk.py

```diff
@@ -42,32 +44,53 @@
 )
 
 
-def get_config_path():
-    """Returns the absolute path the the Cloud SDK's configuration directory.
+def get_config_path(config_directory=_CONFIG_DIRECTORY_GCLOUD):
```

Comment:
A cursory look suggests this is only used in gcloud's code base.

Can you own making sure that there is no regression when this library is updated in Piper?

Reply:
Yes of course. I was actually making a change in the gcloud code base first and that's when I realized the path definitions happen here. How often do you merge into google3 and when will this change be expected to land?

Reply:
The merge is manual. I can provide instructions for how you can do it

Reply:
Hi @clundin25, I've received an approval from @sai-sunder-s and am ready for merge instructions now.